R/internals.R

```diff
@@ -379,3 +379,20 @@
 #   if(getOption("mypackage.verbose", default = TRUE)) message(glue::glue(msg))
 #   return(NULL)
 # }
+
+#' Load Page with headers
+#'
+#' loads webpages with a header passed to read_html
+#'
+#' @param page_url url of the page wanted to be loaded
+#'
+#' @return a html webpage
+#'
+#' @noRd
+#'
+.load_page <- function(page_url) {
+  agent <- getOption("worldfootballR.agent", default = "RStudio Desktop (2022.7.1.554); R (4.1.1 x86_64-w64-mingw32 x86_64 mingw32)")
```

Comment:
The `default` is in case the option doesn't work, but I think it should always work since I think it's impossibly for a user to call a function from a package without `.onLoad()` running.